specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -323,6 +323,267 @@
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "releaseId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Release identifier within an API. Must be unique in the current API Management service instance.",
+            "minLength": 1,
+            "maxLength": 256,
+            "pattern": "^[^*#&+:<>?]+$"   
+          }
+        ],
+        "get": {
+        "tags": [
+          "Releases"
+        ],
+        "operationId": "Api_GetRelease",
+        "description": "Returns the details of an API release.",
+        "responses": {
+          "200": {
+            "description": "The operation returns a list of API Releases.",
+            "schema": {
+              "$ref": "#/definitions/ApiReleaseCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ApiReleaseContract"
+      },
+      "put": {
+        "tags": [
+          "Releases"
+        ],
+        "operationId": "Api_CreateOrUpdateRelease",
+        "description": "Creates a new Release for the API or updates an existing one.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiReleaseContract"
+            },
+            "description": "Create parameters."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Release was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/ApiReleaseContract"
+            }
+          },
+          "200": {
+            "description": "Release was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ApiReleaseContract"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the release failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Releases"
+        ],
+        "operationId": "ApiRelease_Update",
```

Comment:
operationId is how documentation gets generated

"ApiRelease_Update" controls that they create a "Api Release" heading and Update comes as operation.

It helps as 

ApiRelease_List
ApiRelease_Get
ApiRelease_CreateOrUpdate
etc all come in under one section